src/pages/cookie.tsx

```diff
@@ -44,7 +44,7 @@ export default () => (
   <Layout title="Cookie Policy">
     <ContentWrapper>
       <Title>Cookie Policy</Title>
-      <Date>Last updated on April, 2020</Date>
+      <Date>Last updated on February, 2020</Date>
```

Comment:
```suggestion
      <Date>Last updated on February, 2022</Date>
```

Reply:
Do we leave `February` or should it be `March`?

Reply:
Nice catch! Your changes were in February.